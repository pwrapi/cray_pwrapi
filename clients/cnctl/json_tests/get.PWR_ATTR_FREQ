{
 "PWR_Function"          : "PWR_ObjAttrGetValues",
 "PWR_Attrs"             : [
                            { "PWR_AttrName" : "PWR_ATTR_FREQ" }
                           ],
 "PWR_MajorVersion"      : 32,
 "PWR_MinorVersion"      : 67
}
