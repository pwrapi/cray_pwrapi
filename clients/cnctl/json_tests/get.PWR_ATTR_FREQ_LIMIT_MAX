{
 "PWR_Function"          : "PWR_ObjAttrGetValues",
 "PWR_Attrs"             : [
                            { "PWR_AttrName" : "PWR_ATTR_FREQ_LIMIT_MAX" }
                           ],
 "PWR_MajorVersion"      : 32,
 "PWR_MinorVersion"      : 67
}
