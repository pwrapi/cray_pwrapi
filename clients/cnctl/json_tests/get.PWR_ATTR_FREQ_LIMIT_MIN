{
 "PWR_Function"          : "PWR_ObjAttrGetValues",
 "PWR_Attrs"             : [
                            { "PWR_AttrName" : "PWR_ATTR_FREQ_LIMIT_MIN" }
                           ],
 "PWR_MajorVersion"      : 32,
 "PWR_MinorVersion"      : 67
}
