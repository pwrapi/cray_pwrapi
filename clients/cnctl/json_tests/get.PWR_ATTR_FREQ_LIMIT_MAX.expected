{
 "PWR_Attrs"             : [
                            {
                             "PWR_AttrName":"PWR_ATTR_FREQ_LIMIT_MAX",
                             "PWR_AttrValue":2301000,
                             "PWR_TimeSeconds":1426191734,
                             "PWR_TimeNanoseconds":533763402,
                             "PWR_ReturnCode":0
                            }
                           ],
 "PWR_MajorVersion"      : 0,
 "PWR_MinorVersion"      : 1,
 "PWR_ReturnCode"        : 0,
 "PWR_Messages"          : null,
 "PWR_ErrorMessages"     : null
}
